service/tbc/tbc.go

```diff
@@ -609,7 +638,7 @@ func (s *Server) sod(ctx context.Context, p *peer) (*chainhash.Hash, error) {
 		return &bhb.Hash, nil
 	}
 	if bhb.Height > uint64(p.remoteVersion.LastBlock) {
-		log.Debugf("sod: %v our tip is greater %v > %v",
```

Comment:
This was changed from debug to info; was this for debugging, should it be reverted?

Reply:
No this time. Later.